explanations/machine-learning-terms.md

```diff
@@ -25,6 +25,22 @@ Researchers often invent complicated names for simple things. This page will hel
     <p>One example is working with words in Natural Language Processing. Words are usually transformed to a list of numbers  (embedded) before given as input to a neural network. The exact mapping between word and the list of numbers will be automatically learned.</p>
 </td>
 </tr>
+<tr>
+<td valign="top">Principal Component Analysis (PCA)</td>
+<td>
+    <p>An algorithm that is used to narrow down datasets.</p>
+    <p>PCA (as any 'dimensionality reduction' algorithms) has several advantages:
+	<ul>
+		<li>it reduces the storage space required</li>
+		<li>it improves the performance of the machine learning model by removing the multicollinearity in the data</li>
+		<li>it eases the visualization of the data when reduced to very low dimensions such as 2D or 3D</li>
+	</ul>
+    </p>
+    <p>Technically, it's a statistical procedure that uses an orthogonal transformation to <strong>identify a small number of uncorrelated variables, called principal components</strong> from a large set of data possibly containing correlated variables.</p>
```

Comment:
This is still too complicated... I would maybe remove this and jump to the example below, which is good!